<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>800</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>800</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;/home/iot/바탕화면/★★★★★/농산물 가격 데이터 수집 및 시각화/서ㅏ진/1.png&quot;)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2013</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2014</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2022</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>430</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>전라북도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton8">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>490</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>경상남도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton6">
      <property name="geometry">
       <rect>
        <x>389</x>
        <y>330</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>경상북도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>충청북도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>충청남도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>강원도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);

</string>
      </property>
      <property name="text">
       <string>서울특별시</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>경기도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton10">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>700</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>제주특별자치도</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton9">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>540</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>전라남도</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QFrame" name="frame2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>800</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>660</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>최소</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>720</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>평균</string>
      </property>
     </widget>
     <widget class="QLabel" name="max_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>690</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>19</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>상품</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>690</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>최대</string>
      </property>
     </widget>
     <widget class="QLabel" name="min_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>660</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>491</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="graph_verticalLayout"/>
     </widget>
     <widget class="QLabel" name="average_label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>720</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton13">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>40</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;
border: 2px solid rgb(45, 45, 45);
background-color: rgba(191, 64, 64, 0);</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>40</y>
        <width>121</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>열무</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>얼갈이배추</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>물오징어</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>명태</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>갈치</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>고등어</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>방울토마토</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>감귤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>단감</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>바나나</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>참다래</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>팽이버섯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>사과</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>배</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>포도</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>감귤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>파</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>생강</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>미나리</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>깻잎</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>피망(청)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>느타리버섯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>호박</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>토마토</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>무</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>당근</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
